src/main/java/io/aiven/kafka/auth/AivenAclAuthorizer.java

```diff
@@ -40,6 +40,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+@SuppressWarnings("deprecation")
```

Comment:
I think it is better to add `@Deprecated` annotation for this class, just to highlight that we are going to rid of it

Reply:
Valid point, done.